### PR TITLE
Use ts-node fast mode to speed up compilation time

### PR DESCRIPTION
**Pull request by geek-soft**
### JIRA link

???

### Change description

This PR enables ts-node fast mode. That speeds up performance so that compilation with empty cache takes around 7 seconds on my backup machine (slower then yours :P).

I tried ts-node 4.0.1 however it looks like compilation time when that version is used is slightly but noticeable higher. I have impression that ts-node 4.0.1 does not use cache properly which makes each compilation start with empty cache. I need to investigate it more deeply before I raise that issue with ts-node team.

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [x] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
       "es2017.object",
       "dom"
     ],
-    "target": "es5",
+    "target": "es6",
```
Comment (geek-soft) [inline tsconfig.json]: Not strictly necessary however ES6 code is smaller then ES5 (2.8MB vs 3.1MB) so your hard drives won't work that hard and you probably save 1ms here :)

**Comment by geek-soft** [comment]
Either as a part of this PR or in separate PR I will remove remove TS>JS compilation. It will make startup time longer by around 4 seconds on my machine but we will get proper stack traces back.

File: package.json [hunk 2]
```diff
@@ -10,14 +10,14 @@
     "copy-assets": "gulp sass copy-files",
     "setup": "yarn compile && yarn copy-assets",
     "start": "node src/main/server.js",
-    "start-dev": "./node_modules/.bin/ts-node src/main/server.ts",
+    "start-dev": "TS_NODE_FAST=true ./node_modules/.bin/ts-node src/main/server.ts",
```
Comment (timja-kainos) [inline package.json]: not planning on upgrading back to 4.0?

Comment (geek-soft) [reply package.json]: Not now due to the fact that event if we did not have an issue with nodemon compilation in ts-node 4.0 it is still slower. I will raise issues with them and let them fix it.

**Pull request merged**